src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -0,0 +1,135 @@
+package org.allenai.plugins
+
+import sbt.{
+  relativeTo,
+  settingKey,
+  taskKey,
+  AutoPlugin,
+  Def,
+  Keys,
+  PathFinder,
+  Plugins,
+  SettingKey,
+  TaskKey
+}
+import sbt.plugins.JvmPlugin
+
+import java.io.File
+
+/** Plugin for building docker images. */
+object DockerBuildPlugin extends AutoPlugin {
+  val AI2_PRIVATE_REGISTRY = "allenai-docker-private-docker.bintray.io"
+
+  val DEFAULT_BASE_IMAGE = AI2_PRIVATE_REGISTRY + "/java:8"
```

Comment:
True.  I think the main thing I'm looking for is that if I forget to add the registry I get a failure instead of pulling the wrong image.